sway/commands/move.c

```diff
@@ -248,12 +248,14 @@ static void container_move_to_container(struct sway_container *container,
 		ipc_event_window(container, "move");
 	}
 
-	workspace_focus_fullscreen(destination->workspace);
+	if (destination->workspace) {
+		workspace_focus_fullscreen(destination->workspace);
 
-	// Update workspace urgent state
-	workspace_detect_urgent(destination->workspace);
-	if (old_workspace && old_workspace != destination->workspace) {
-		workspace_detect_urgent(old_workspace);
+		// Update workspace urgent state
+		workspace_detect_urgent(destination->workspace);
+		if (old_workspace && old_workspace != destination->workspace) {
```

Comment:
This if block should not be inside the `if (destination->workspace)` block. Urgency should still be updated for `old_workspace` (if not `NULL`) even when `destination->workspace` is `NULL`

Reply:
Yep. Thanks.